specification/src/main/asciidoc/platform/ResourcesNamingInjection.adoc

```diff
@@ -209,12 +210,12 @@ shared with any other component. Components in a web module do not have
 their own private component namespace. __ See note below.
 *  _java:module_ – Names in this namespace are
 shared by all components in a module (for example, all enterprise beans
-in a single EJB module, or all components in a web module). __
+in a single Jakarta Enterprise Beans module, or all components in a web module). __
```

Comment:
since we used "web module" in this statement, I would probably use "enterprise beans" module...

Reply:
sure, but I'm going to use enterprise bean, I think it flows better. You made this comment so I did a search and replace for Jakarta Enterprise Beans module and replaced it with enterprise bean module.